src/codeenginestudio/huong/Exercise1.java

```diff
@@ -0,0 +1,47 @@
+package codeenginestudio.huong;
+
+import java.util.Arrays;
+
+public class Exercise1 {
+    public static void main(String[] args) {
+
+        String[] strArray = {"is2", "pa6" ,"Thi1s", "T4est", "3a", "th5"};
```

Comment:
As I read from the requirement, they want to make a function with input is a String, and output also is a String.
IMO, we should have a method with parameter is a String and return type is a String. It makes easy to understand what we need to do.